openregister-conformance

```diff
@@ -20,8 +22,10 @@ pytest_args = [base_dir, '-q']
 if args.no_https:
     pytest_args += ['-m','not https']
 
-for register in args.endpoint:
-    print(register)
-    pytest_args += ['--endpoint', register]
+for register_base in args.endpoint:
```

Comment:
this for loop doesn't need to be a for loop any more if `args.endpoint` no longer allows a list of arguments